e2e/error_1/error_1.spec.js

```diff
@@ -1,30 +1,34 @@
 const { Test, Config } = require('../../src/index');
-const pumpTests = require('../../tests/errors/pump.tests');
+const cgmTests = require('../../tests/errors/cgm.tests');
 
 describe('errors', () => {
     var test = new Test();
     var config = new Config();
-    it('prepare test', async () => {
+    var cgmData;
+    beforeAll(async () => {
         config = await config.prepare();
-        var cgmText = config.text.device.CGMSimulatorScreen;
+        let cgmText = config.text.device.CGMSimulatorScreen;
+        cgmData = {
+            model: { name: cgmText.Model.Constant, bgValues: [97] },
+            frequency: { seconds: true },
+            history: { name: cgmText.History.BackfillGlucose, backfillHours: 5 },
+        };
+    });
+    it('prepare test', async () => {
         test = test.setup({
             language: config.text,
             screenDefaults: config.screenDefaults,
             limits: config.limits,
-            settingDefault: config.settingDefault,
             authentication: { faceid: true },
             enableClosedLoop: true,
+            warmupPeriod: { milliseconds: 10000 },
             enableTherapySettings: true,
             simulators: { cgm: true, pump: true },
-            cgmData: {
-                model: { name: cgmText.Model.Constant, bgValues: [101] },
-                frequency: { seconds: true },
-                history: { name: cgmText.History.BackfillGlucose, backfillHours: 6 },
-            }
+            cgmData: cgmData
         });
         await test.prepare();
     });
-    describe('from pump', () => {
-        pumpTests(test);
+    describe('from cgm', () => {
+        cgmTests(test, cgmData);
```

Comment:
[nit] Doesn't the test already have the `cgmData`?

Reply:
🤦 yup ... have updated thanks